config/application.rb

```diff
@@ -48,6 +49,18 @@ class Application < Rails::Application
       config.i18n.available_locales = [locale].flatten
     end
 
+    # Cache configuration
+    sidekiq_config_for_env = File.join(Rails.root, 'config', "sidekiq-#{Rails.env}.yml")
+    sidekiq_config = File.exist?(sidekiq_config_for_env) ? sidekiq_config_for_env : File.join(Rails.root, 'config', "sidekiq.yml")
+
+    if File.exist?(sidekiq_config) && !Rails.env.test?
+      require 'sidekiq/middleware/i18n'
+      redis_config = YAML.load_file(sidekiq_config)
+      redis_url = { host: redis_config[:redis_host], port: redis_config[:redis_port], db: redis_config[:redis_database], namespace: "cache_checkapi_#{Rails.env}" }
+      config.cache_store = :redis_cache_store, redis_url
```

Comment:
Since we have it here, do we still need it at https://github.com/meedan/check-api/blob/CV2-3055-use-config-environments-production-rb-from-file-not-from-ssm-in-check-api/config/initializers/03_sidekiq.rb#L28 ?

It's a bit awkward that the Redis configuration lives in the Sidekiq configuration file while we use Redis for other stuff... I know it's not in the scope of this ticket, but reviewing this made me realize that we may need to refactor this in a separate ticket (isolate Redis configuration from Sidekiq and use it for both caching and Sidekiq).